From 6dd82c05340b3b80ac765e41f1bb7414be8cf094 Mon Sep 17 00:00:00 2001
From: Aylei <rayingecho@gmail.com>
Date: Sat, 16 Jan 2021 21:04:36 +0800
Subject: [PATCH] Scalffold

Signed-off-by: Aylei <rayingecho@gmail.com>
---
 src/server/status_server/mod.rs | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/server/status_server/mod.rs b/src/server/status_server/mod.rs
index 52391ff07..84fe9f8f2 100644
--- a/src/server/status_server/mod.rs
+++ b/src/server/status_server/mod.rs
@@ -331,6 +331,11 @@ where
             .build()
     }
 
+    pub async fn my_handler(_req: Request<Body>) -> hyper::Result<Response<Body>> {
+        let msg = "TODO";
+        Ok(Response::new(Body::from(format!("{}", msg))))
+    }
+
     pub async fn dump_rsperf_to_resp(req: Request<Body>) -> hyper::Result<Response<Body>> {
         let query = match req.uri().query() {
             Some(query) => query,
@@ -747,6 +752,9 @@ where
                             (Method::GET, "/debug/pprof/profile") => {
                                 Self::dump_rsperf_to_resp(req).await
                             }
+                            (Method::GET, "/hack") => {
+                                Self::my_handler(req).await
+                            }
                             (Method::GET, path) if path.starts_with("/region") => {
                                 Self::dump_region_meta(req, router).await
                             }
-- 
2.27.0

